src/org-units-selector/OrgUnitsSelector.jsx

```diff
@@ -1,4 +1,4 @@
-import { Card, CardContent } from "@material-ui/core";
```

Comment:
This file is not prettified (`yarn prettify`). Having vscode with extension `Prettier` + setting "Format on save" enabled should solve the problem altogether.

Reply:
Right, and set it as the default formatter in the general preferences. I changed my programming setup and missed it, even though you had already showed me this. Fixed now. Thanks!